templates/dashboard/api_key.html

```diff
@@ -22,6 +22,16 @@ <h1 class="h3"> API Keys </h1>
         ️API Keys should be kept secret and treated like passwords, they can be used to gain access to your account.
       </div>
 
+      {% if api_keys|length > 0 %}
```

Comment:
I'd suggest to move the button to the bottom of the page to not "encourage" users to use it immediately when they arrive to the page.

Reply:
Yup, I thought about doing this but thought it'd be a bit hard to find if you had a load of API Keys, but I guess the majority of users have only a few.